core/src/autogluon/core/trainer/abstract_trainer.py

```diff
@@ -189,13 +194,15 @@ def load_model(self, model_name: str | ModelTypeT, path: str | None = None, mode
             return self.models[model_name]
         else:
             if path is None:
-                path = self.get_model_attribute(model=model_name, attribute="path")  # get relative location of the model to the trainer
+                path = self.get_model_attribute(
+                    model=model_name, attribute="path"
+                )  # get relative location of the model to the trainer
                 assert path is not None
             if model_type is None:
                 model_type = self.get_model_attribute(model=model_name, attribute="type")
                 assert model_type is not None
             return model_type.load(path=os.path.join(self.path, path), reset_paths=self.reset_paths)
-        
```

Comment:
Maybe open a separate PR first to enable formatting for core/tabular with `ruff`? This would simplify the reviewing imo.

Reply:
agreed and done 👍

Reply:
i.e., reverted the formatting-only changes for now.